File: Selenium/Activities/Activity12.java
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity12 {
    public static void main(String[] args) {
        WebDriver driver = new FirefoxDriver();

        driver.get("https://v1.training-support.net/selenium/dynamic-controls");
        System.out.println(driver.getTitle());

        boolean x = driver.findElement(By.xpath("//*[@id=\"input-text\"]")).isEnabled();
        System.out.println(x);

        driver.findElement(By.xpath("//*[@id=\"toggleInput\"]")).click();

        System.out.println(driver.findElement(By.xpath("//*[@id=\"input-text\"]")).isEnabled());

        driver.quit();
    }
}

File: Selenium/Activities/Activity12.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.support.wait import WebDriverWait
from selenium.webdriver.support import expected_conditions as EC

driver = webdriver.Firefox()
wait = WebDriverWait(driver, 20)

driver.get("https://v1.training-support.net/selenium/dynamic-controls")
print(driver.title)

Bool = driver.find_element(By.XPATH, '//*[@id="input-text"]').is_enabled()
print(Bool)

driver.find_element(By.XPATH, '//*[@id="toggleInput"]').click()

print(driver.find_element(By.XPATH, '//*[@id="input-text"]').is_enabled())

driver.quit()
